Makefile

```diff
@@ -37,8 +37,30 @@ build-sdk-wasm-pkg:
 	@cd crates/jstz_sdk && wasm-pack build --target bundler --release
 
 .PHONY: test
```

Comment:
We might want to call ligo test here too

Reply:
I'm purposely avoiding LIGO for now, since it is tricky to get the dependency into our CI